.rubocop.yml

```diff
@@ -44,9 +45,13 @@ Metrics/MethodLength:
     - lib/iron_bank/resources/product_rate_plan_charge_tier.rb
     - lib/iron_bank/error.rb
     - lib/iron_bank/client.rb
+    - lib/iron_bank/describe/excluded_fields/deduce_from_query.rb
 
 Style/ClassAndModuleChildren:
   EnforcedStyle: nested
+  Exclude:
```

Comment:
is it possible to eliminate these without moving away from the codebase's general layout/style?

Reply:
Done